content/applications/general/calendars/google/google_calendar_credentials.rst

```diff
@@ -0,0 +1,155 @@
+
+Next, if continuing in testing mode (External), add the email addresses being configured under the
+:guilabel:`Test users` step, by clicking on :guilabel:`Add Users`, and then the :guilabel:`Save and
+Continue` button. A summary of the :guilabel:`App registration` appears.
+
+Finally, scroll to the bottom, and click on :guilabel:`Back to Dashboard`.
+
+Now, the OAuth consent has been configured, and it's time to create credentials.
+
+Create credentials
+------------------
+
+The **Client ID** and the **Client Secret** are both needed to connect Google Calendar to Odoo. This
+is the last step in the Google console. Begin by clicking :guilabel:`Credentials` in the left menu.
+Then, click :guilabel:`Create Credentials`, and select :guilabel:`OAuth client ID`, Google will open
+a guide to create credentials.
+
+Under :guilabel:`OAuth Client ID`, select :guilabel:`Website application` for the
+:guilabel:`Application Type` field, and type *My Odoo Database* for the :guilabel:`Name`.
+
+  - Under the :guilabel:`Authorized JavaScript Origins` section, click :guilabel:`+ Add URI` and
+    type the company's Odoo full :abbr:`URL (Uniform Resource Locator)` address.
+
+  - Under the :guilabel:`Authorized redirect URIs` section, click :guilabel:`+ Add URI` and type
+    the company's Odoo :abbr:`URL (Uniform Resource Locator)` address followed by
+    ``/google_account/authentication``. Finally, click :guilabel:`Create`.
+
+.. image:: google_calendar_credentials/uri.png
+   :align: center
+   :alt: Add the authorized JavaScript origins and the authorized redirect URIs.
+
+A :guilabel:`Client ID` and :guilabel:`Client Secret` will appear, copy these to a notepad.
+
+Setup in Odoo
+=============
+
+Once the Client ID and the Client Secret are located, open the Odoo database and go to
+:menuselection:`Settings --> General Settings --> Integrations --> Google Calendar`. Check the box
+next to :guilabel:`Google Calendar`.
+
+.. image:: google_calendar_credentials/settings-google-cal.png
+   :align: center
+   :alt: The Google Calendar checkbox in General Settings.
+
+Next, copy and paste the Client ID and the Client Secret from the Google Calendar API Credentials
+page into their respective fields below the :guilabel:`Google Calendar` checkbox. Then, click
+:guilabel:`Save`.
+
+Sync Calendar in Odoo
+=====================
+
+Finally, open the Calendar module in Odoo and click on the :guilabel:`Google` sync button to sync
+Google Calendar with Odoo.
+
+.. image:: google_calendar_credentials/sync-google.png
+   :align: center
+   :alt: Click the Google sync button in Odoo Calendar to sync Google Calendar with Odoo.
+
+.. note::
+   When syncing Google Calendar with Odoo for the first time, the page will redirect to the Google
+   Account. From there, select the :menuselection:`Email Account` that should have access -->
+   :menuselection:`Continue` (should the app be unverifed) --> :menuselection:`Continue` (to give
+   permission for the transfer of data).
```

Comment:
The phrasing and formatting within this paragraph makes supporting the proper :menuselection: syntax difficult, I'm suggesting we rephrase it to simply use :guilabels:

```
When syncing Google Calendar with Odoo for the first time, the page will redirect to the Google Account. From there, select the :guilabel:`Email Account` that should have access, then select :guilabel:`Continue` (should the app be unverifed), and finally select :guilabel:`Continue` (to give permission for the transfer of data)`.
```